README.md

```diff
@@ -148,6 +148,13 @@ Next, provide the path to the `subnet-evm` repository and the path to a writeabl
 ```bash
 ./scripts/e2e_test.sh --local --subnet-evm ~/subnet-evm --data-dir ~/tmp/e2e-test
 ```
+
+The E2E tests use the `TeleporterMessenger` contract deployment transaction specified in the following files:
+- `tests/utils/UniversalTeleporterDeployerAddress.txt`
+- `tests/utils/UniversalTeleporterDeployerTransaction.txt`
+- `tests/utils/UniversalTeleporterMessagerContractAddress.txt`
+To update the version of Teleporter used by the E2E tests, update these values with the latest contract deployment information.
```

Comment:
Can you link to the deployment utils README in `teleporter`?

Reply:
This [contract deployment link](https://github.com/ava-labs/teleporter/blob/main/contract-deployment/README.md) is not found. Where should I link in `teleporter`?

Reply:
https://github.com/ava-labs/teleporter/tree/main/utils/contract-deployment

Reply:
added the link